stalker_ui/src/theme.scss

```
@@ -0,0 +1,75 @@
+@use '@angular/material' as mat;
+@import './palette.scss';
+
+@include mat.core();
+
+:root {
+  --density-scale: -2;
+  --small-gap-size: calc(16px + var(--density-scale) * 4px);
+  --normal-gap-size: calc(24px + var(--density-scale) * 4px);
+}
+
+$primary: mat.define-palette($primary, 500);
+$accent: mat.define-palette($warn, A200, A100, A400);
+
+$typography: mat.define-typography-config(
+  $body-1:
+    mat.define-typography-level(
+      $font-size: 14px,
+      $line-height: 24px,
+    ),
+  $button:
+    mat.define-typography-level(
+      $font-size: 13px,
+    ),
+  $caption:
+    mat.define-typography-level(
+      $font-size: 11px,
+      $line-height: 11px,
+    ),
+);
+
+$stalker-light: mat.define-light-theme(
+  (
+    color: (
+      primary: $primary,
+      accent: $accent,
+    ),
+    density: -2,
+    typography: $typography,
+  )
+);
```

Je pense qu'il manquait ça ici:

```suggestion
$stalker-light: mat.define-light-theme(
  (
    color: (
      primary: $primary,
      accent: $accent,
      warn: $warn
    ),
    density: -2,
    typography: $typography,
  )
);
```